### PR TITLE
use temporally_independent_train_step for sac_carla_conf.py

### PR DESCRIPTION
**Pull request by emailweixu**

### CODE REVIEW

File: alf/examples/sac_carla_conf.py [hunk 1]
```diff
@@ -187,14 +176,33 @@
...
     value_net_cls = partial(
         alf.networks.ValueNetwork,
         fc_layer_params=(256, ),
         output_tensor_spec=env.reward_spec())
 
     alf.config('Agent', rl_algorithm_cls=TasacValueAlgorithm)
+    alf.config(
+        'TrainerConfig',
+        use_rollout_state=True,
+        temporally_independent_train_step=True)
```
Comment (hnyu) [inline alf/examples/sac_carla_conf.py:204]: Is this the reason why the current code is much slower than before?

File: alf/examples/sac_carla_conf.py [hunk 2]
```diff
@@ -187,14 +176,33 @@
...
+
     alf.config(
         'TasacValueAlgorithm',
         actor_network_cls=actor_network_cls,
```
Comment (hnyu) [inline alf/examples/sac_carla_conf.py:208]: Line 214 "always use 1-step TD" should have been removed

File: alf/examples/sac_carla_conf.py (hunk 1 above)
Comment (emailweixu) [reply alf/examples/sac_carla_conf.py:204]: yes

File: alf/examples/sac_carla_conf.py (hunk 2 above)
Comment (emailweixu) [reply alf/examples/sac_carla_conf.py:208]: removed

**Review by hnyu — APPROVED** [review]

**Pull request merged**